### ISSUE TITLE
Updating setting constants from outside

### ISSUE THREAD

**Issue by cyberwani** [Jhut89/Mailchimp-API-3.0-PHP#13]
Hello,

I am trying to update settings from where this library is called.

```php
<?php
include('mailchimp-api-3.0-php/vendor/autoload.php');

$api_key = '123abc123abc123abc123abc-us0';
$list_id = '132465789';

if (!defined('VERIFY_SSL')) {
    define('VERIFY_SSL', false);
}

$mailchimp = new Mailchimp($api_key);

$post_params = array(
	'email_address' => 'email@example.com',
	'status'        => 'subscribed'
);
$response = $mailchimp->lists($list_id)->members()->POST($post_params);
```
So, if constants can be updated from construtor, it can be more helpful.

**Comment by Jhut89** [comment]
I can definitely see the utility in being able to edit these programatically. I can look into an implementation. That said this would likely be a move away from `constants` since these sort-of imply they are not changeable at run time.

**Comment by Jhut89** [comment]
Addressed in 3.0.0
